cmd/agent/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM centos:7
```

Comment:
The images are inheriting the CentOS 7 images, as those provide some tools which are useful for debugging. Once the images mature a bit on the community, we can change the base image to Alpine or similar.